geocoding/generate_pincode.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import math
+import pathlib
+
+import fire
+import googlemaps
+import pandas as pd
+from tqdm import tqdm
+
+logging.basicConfig(
+    filename="pincoding.log",
+    filemode="a",
+    format="%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s",
+    datefmt="%H:%M:%S",
+    level=logging.INFO,
+)
+
+logging.info("Logger Setup Complete")
```

Comment:
Maybe move the logging config to a separate file and use that?

Maybe consider using Logaru? https://github.com/Delgan/loguru
Or Structlog -- which is what we use at @verloop under the hood behind Verloop-CTL?

Reply:
Shifted the config to another file.
Switched to loguru.